app/src/main/java/org/jellyfin/mobile/player/source/MediaSourceResolver.kt

```diff
@@ -34,6 +34,7 @@ class MediaSourceResolver(
                     audioStreamIndex = audioStreamIndex,
                     subtitleStreamIndex = subtitleStreamIndex,
                     maxStreamingBitrate = /* 1 GB/s */ 1_000_000_000,
+                    autoOpenLiveStream = true
```

Comment:
Please use trailing commas when possible.
```suggestion
                    autoOpenLiveStream = true,
```

Reply:
Thanks for the quick review. I added the comma as requested.

Additionally, I looked at processes on both Android and the server to confirm there are no orphaned processes after video stops. I also confirmed the tuner no longer has any active sessions.

The following log items appear in my server log, which validates the code I saw in the repository last night. As you can see, once the app notifies the server that playback has stopped, the server determines the consumer count is zero and performs the necessary cleanup of the SharedHttpStream with the tuner and cancels/deletes the transcoding session.

> [2021-08-18 01:01:06.901 +00:00] [INF] [11] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Android" "0.0.0-dev.1" playing "KSHB-TV". Stopped at "18165" ms
[2021-08-18 01:01:06.907 +00:00] [INF] [38] Emby.Server.Implementations.Library.MediaSourceManager: Live stream "native_686b312d3504901ba64fed43701d1733_9219b79e0dbf2b42af414df829c1f675" consumer count is now 0
[2021-08-18 01:01:06.907 +00:00] [INF] [38] Emby.Server.Implementations.Library.MediaSourceManager: Closing live stream "a17c75760a04e99b68cf766e11316e1c_09efa0d56b934a82adec00a87b837fb0_native_686b312d3504901ba64fed43701d1733_9219b79e0dbf2b42af414df829c1f675"
[2021-08-18 01:01:06.907 +00:00] [INF] [38] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Closing "SharedHttpStream"
[2021-08-18 01:01:06.907 +00:00] [INF] [38] Emby.Server.Implementations.Library.MediaSourceManager: Live stream "a17c75760a04e99b68cf766e11316e1c_09efa0d56b934a82adec00a87b837fb0_native_686b312d3504901ba64fed43701d1733_9219b79e0dbf2b42af414df829c1f675" closed successfully
[2021-08-18 01:01:07.220 +00:00] [INF] [22] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Copying of "SharedHttpStream" to "/config/transcodes/e75e9b1e6025408bb7245eb5aacb31c0.ts" was canceled
[2021-08-18 01:01:07.221 +00:00] [INF] [22] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Deleting temp files ["/config/transcodes/e75e9b1e6025408bb7245eb5aacb31c0.ts"]
[2021-08-18 01:01:17.691 +00:00] [INF] [35] Jellyfin.Api.Helpers.TranscodingJobHelper: Transcoding kill timer stopped for JobId "8554f07b8e234f84b35683592e405f7d" PlaySessionId "bb5fb5c834044d7f9b571f2582e4ae8a". Killing transcoding
[2021-08-18 01:01:17.692 +00:00] [INF] [35] Jellyfin.Api.Helpers.TranscodingJobHelper: Stopping ffmpeg process with q command for "/config/transcodes/c7e30cf6f09a787f455aea00a0661fc7.m3u8"
[2021-08-18 01:01:22.692 +00:00] [INF] [35] Jellyfin.Api.Helpers.TranscodingJobHelper: Killing FFmpeg process for "/config/transcodes/c7e30cf6f09a787f455aea00a0661fc7.m3u8"
[2021-08-18 01:01:22.692 +00:00] [INF] [35] Jellyfin.Api.Helpers.TranscodingJobHelper: Deleting partial stream file(s) "/config/transcodes/c7e30cf6f09a787f455aea00a0661fc7.m3u8"
[2021-08-18 01:01:22.703 +00:00] [ERR] [22] Jellyfin.Api.Helpers.TranscodingJobHelper: FFmpeg exited with code 137